dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -153,12 +153,7 @@ def exit_survey(enrollment)
     @workshop = enrollment.workshop
     @teacher = enrollment.user
     @enrollment = enrollment
-
-    if [Pd::Workshop::COURSE_ADMIN, Pd::Workshop::COURSE_COUNSELOR].include? @workshop.course
-      @survey_url = CDO.code_org_url "/pd-workshop-survey/counselor-admin/#{enrollment.code}", 'https:'
-    else
-      @survey_url = CDO.code_org_url "/pd-workshop-survey/#{enrollment.code}", 'https:'
-    end
+    @survey_url = enrollment.exit_survey_url
```

Comment:
This refactor is not directly related, but since I was messing with the exit survey I moved this definition into `Pd::Enrollment` and added a unit test.